topostats/utils.py

```diff
@@ -204,7 +205,7 @@ def get_thresholds(
     image: np.ndarray,
     threshold_method: str,
     otsu_threshold_multiplier: float = None,
-    threshold_std_dev: float = None,
+    threshold_std_dev: tuple = None,
```

Comment:
Do you mean change it in the config file so that instead of:

```
threshold_std_dev_upper: 1.0
threshold_std_dev_lower: 1.0
```

we have 

```
threshold_std_dev: [1.0, 1.0]
```

if so, should this change also be made for `threshold_absolute_upper` and `threshold_absolute_lower`? 

I think that it might be better to leave the values in the config file as separate lines to maintain simplicity (not having to figure out which is upper and lower) though I'm very happy to have pushback on this - just a very minor preference.

Reply:
I kind of like dictionaries for this sort of thing as the simplify usage and keep things clear.

And so in `topostats/default_config.yaml` we can have...

```
threshold_std_dev:
  lower: 1.0
  upper: 1.0
```

As passing a dictionary(/list/tuple) reduces the number of options to a class/argument and avoids hitting the linting error [`too-many-arguments` / `R0913`](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/too-many-arguments.html).

Reply:
Added 👍